mycore-pi/src/main/java/org/mycore/pi/urn/rest/MCRDNBURNRestClient.java

```diff
@@ -18,47 +18,107 @@
 
 package org.mycore.pi.urn.rest;
 
-import static org.apache.http.entity.ContentType.APPLICATION_XML;
-
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
 import java.net.URL;
 import java.time.Instant;
 import java.time.format.DateTimeFormatter;
 import java.util.Date;
 import java.util.Optional;
 import java.util.function.Function;
 
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParseException;
+import com.google.gson.JsonParser;
 import org.apache.http.Header;
 import org.apache.http.HttpStatus;
 import org.apache.http.StatusLine;
+import org.apache.http.auth.UsernamePasswordCredentials;
 import org.apache.http.client.methods.CloseableHttpResponse;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.mycore.common.config.MCRConfiguration2;
 import org.mycore.pi.MCRPIRegistrationInfo;
 
+import static org.apache.http.entity.ContentType.APPLICATION_JSON;
+
 /**
+ * Class for registering and updating urn managed by the DNB.
+ * @see <a href="https://api.nbn-resolving.org/v2/docs/index.html">URN-Service API</a>
+ * <br/> <br/>
```

Comment:
this leads to javadoc errors
Error:  /home/runner/work/mycore/mycore/mycore-pi/src/main/java/org/mycore/pi/urn/rest/MCRDNBURNRestClient.java:49: error: self-closing element not allowed
Error:   * <br/> <br/>